### ISSUE TITLE
How to also change the name displayed in task manager?

### ISSUE THREAD

**Issue by dbsxdbsx** [Syed-Waleed-Shah/rename_app#2]
First, thanks for your package, which is convenient to change app name for all platform.

But there’s a fly in the ointment:  the name displayed in the task manager is still not changed---I tested on android and win10.

**Comment by Syed-Waleed-Shah** [comment]
Thanks for compliment. Can you please share the screenshot? I am a
bit confused regarding this. Are you talking about flutter web specifically
or it's a general issues? A screenshot will be much helpful to dig out the
issue.

**Comment by dbsxdbsx** [comment]
I need to build an app that would be used for android and win10, not web.  
For Win10, I mean this, you can get the window by right clicking tool bar with hot key "K":
![f2](https://user-images.githubusercontent.com/17372655/201580724-08c58e97-32c1-4357-8ae5-df13edf0bc72.png)
For Android, I mean this (it would show by tapping overview button on mobile phone):
![fff](https://user-images.githubusercontent.com/17372655/201580617-daf6826c-d302-4686-a4ab-bbd075285329.png)

**Comment by Syed-Waleed-Shah** [comment]
This is not a display name. This is app name and you should change it the code. To do so goto your root widget where you defined MaterialApp(). In the material app there is a property called 'title'. Set the value of 'title' as you want.
Here is an example:
```
return MaterialApp(
      debugShowCheckedModeBanner: false,
      title: 'Your App Title', // Here you can set app title
      home: Scaffold(
        appBar: AppBar(
          title: Text('Home Page'),
        ),
        body: HelloRectangle(),
      ),
    );
```
Important: @dbsxdbsx  If you got your answer then please close the issue and mark resolved.

**Comment by dbsxdbsx** [comment]
> This is not a display name. This is app name and you should change it the code. To do so goto your root widget where you defined MaterialApp(). In the material app there is a property called 'title'. Set the value of 'title' as you want. Here is an example:
> 
> ```
> return MaterialApp(
>       debugShowCheckedModeBanner: false,
>       title: 'Your App Title', // Here you can set app title
>       home: Scaffold(
>         appBar: AppBar(
>           title: Text('Home Page'),
>         ),
>         body: HelloRectangle(),
>       ),
>     );
> ```
> 
> Important: @dbsxdbsx If you got your answer then please close the issue and mark resolved.

This approach partly solved my issue---It is workable on Android, but not win10.
(I would close this issue soon, if I can't find a workaround on Windows)

**Comment by dbsxdbsx** [comment]
Ok, now I found a workaround on win10.
On Win10, to change the  process name shown in task manager, a workaround is to change the description name of file `windows\runner\Runner.rc` :
```rc
 BEGIN
            VALUE "CompanyName", "com.example" "\0"
            VALUE "FileDescription", "name_shown_in_task_manager" "\0"
            VALUE "FileVersion", VERSION_AS_STRING "\0"
...
```
Changing the `FileDescription` value would work the issue out.
Besides, on Win10,  there is also another name (called taskbar name?) shown  in pop up menu when right click the running app on taskbar . To work it out, what I found is to change the `regedit.exe`. The detailed solution is [here](https://stackoverflow.com/questions/65733963/change-taskbar-name-of-flutter-app-on-windows).
Besides, to change the main window title on win10, [cpp file](https://stackoverflow.com/questions/64800455/how-to-change-app-icon-and-app-name-for-flutter-desktop-application#:~:text=To%20change%20application%20icon%3A%20Simply,your%20application%20name%20inside%20window.) should be changed accordingly, which is different from that of Android.

Hopefully, the workaround could be done with a command in `rename_app`.

But, indeed, what I can't help but wonder is why Flutter does not officially provide a built-in command to do the renaming things---After all, this is a quite common demand.
